### ISSUE TITLE
versions doesn't align to request url

### ISSUE THREAD

**Issue by dovholuknf** [openziti/ziti#1188]
## Expected Response:

I expected the url returned to be consistent with the URI I was accessing /versions at

## Actual Response:

The response contains only the 'advertised address' configured in the controller. Although I curl to `ctrl.clint.demo.openziti.org`, the response contains: `ec2-13-58-222-94.us-east-2.compute.amazonaws.com`

curl https://ctrl.clint.demo.openziti.org:8441/
```
{"data":{"apiVersions":{"edge":{"v1":{"apiBaseUrls":["https://ec2-13-58-222-94.us-east-2.compute.amazonaws.com:8441/edge/client/v1"],"path":"/edge/client/v1"}},"edge-client":{"v1":{"apiBaseUrls":["https://ec2-13-58-222-94.us-east-2.compute.amazonaws.com:8441/edge/client/v1"],"path":"/edge/client/v1"}},"edge-management":{"v1":{"apiBaseUrls":["https://ec2-13-58-222-94.us-east-2.compute.amazonaws.com:8441/edge/management/v1"],"path":"/edge/management/v1"}}},"buildDate":"2020-01-01 01:01:01","revision":"local","runtimeVersion":"go1.20.5","version":"v0.0.0"},"meta":{}}
```

**Comment by qrkourier** [comment]
I may have assumed incorrectly but I think the purpose of those links is for API discovery by clients, so they'd need to report the correct, advertised addresses that API clients should use for further operations.

**Comment by bestrocker221** [comment]
I am struggling with that as well. Is it possible to change the apiBaseUrls?

In my current docker compose setup, I could not find any documentation or info on how to specify the apiBaseUrls.
Now I fear I have to redeploy and reconfigure from scratch, right?

**Comment by dovholuknf** [comment]
No, I wouldn't think you have to redeploy. You must have setup alt server certs, like in my case? I did it for browzer last year so in my case both these curls return the _exact_ same response and both are fine/valid:
* curl https://ctrl.clint.demo.openziti.org:8441/
* curl -sk https://ec2-3-142-245-63.us-east-2.compute.amazonaws.com:8441

The bug is minor, in my opinion, and is because the controller is returning a response that worked before we had SNI/alternative cert support. It shouldn't cause any problems.

What sort of issue are you experiencing or worried about?
